test/appium/tests/atomic/transactions/test_wallet.py

```diff
@@ -487,9 +489,12 @@ def test_send_funds_between_accounts_in_multiaccount_instance(self):
         send_transaction.back_button.click()
         balance_of_sub_account = float(self.network_api.get_balance(sub_account_address)) / 1000000000000000000
         balance_of_status_account = float(self.network_api.get_balance(status_account_address)) / 1000000000000000000
-        expected_balance = str(float(balance_after_receiving_tx) - transaction_amount_1 - float(total_fee))
+        self.driver.implicitly_wait(60)
```

Comment:
No need in implicit wait if we confirm (by api_request) that status_account_address received transaction from sub_account. For this we need fix above two lines on to:
        self.network_api.wait_for_confirmation_of_transaction(status_account_address, transaction_amount_1)
        self.network_api.verify_balance_is_updated((updated_balance+transaction_amount_1), status_account_address)

Reply:
Added! Thx a lot.